### DISCUSSION TITLE
Push Notification not working for ios when build is made from flutterflow codemagic deployment

### DISCUSSION THREAD

**Discussion by SpeedotrackGPS** [codemagic-ci-cd/codemagic-docs#2607] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Push Notification not working for ios but working for android when build is made from flutter-flow code magic deployment

### Expected results

Push notification to work in ios device by making build from codemagic

### Actual results

Push notification not working in ios

**Comment by himesh-cm** [comment]
Hey @SpeedotrackGPS,
There could be multiple reasons. To further investigate the issue, consider integrating error monitoring tools like Sentry or Crashlytics, which may help.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.

**Comment by SpeedotrackGPS** [comment]
Hello

But my issue is not resolved when I am building with macbook with Flutter
and xcode I have to enable Push Notification and Background Remote Push
then it is working fine in ios the notification works but when deployed
from Flutter Flow codemagic deployment it is not working

Shubham
